.github/workflows/tests-application.yml

```
@@ -0,0 +1,102 @@
+
+      WORDPRESS_ADMIN_LOGIN:
+        required: true
+      WORDPRESS_ADMIN_PASSWORD:
+        required: true
+      WORDPRESS_ADMIN_EMAIL:
+        required: true
+
+    inputs:
+      WORDPRESS_ARG_BASE_IMAGE:
+        type: string
+        default: 'wordpress:latest'
```

actionlint (1.7.8)

input "WORDPRESS_ARG_BASE_IMAGE" of workflow_call event has the default value "wordpress:latest", but it is also required. if an input is marked as required, its default value will never be used

(events)